alpha/apps/kaltura/lib/webservices/kSessionUtils.class.php

```diff
@@ -580,7 +580,52 @@ public function getDisableEntitlementForEntry()
 		
 		return $entries;
 	}
-	
+
+	public function getDisableEntitlementForPlaylistEntries()
+	{
+		$entries = array();
+		// foreach privileges group
+		foreach( $this->parsedPrivileges as $privilegeType => $privileges)
+		{
+			if ($privilegeType == self::PRIVILEGE_DISABLE_ENTITLEMENT_FOR_PLAYLIST)
+			{
+				foreach($privileges as $privilege)
```

Comment:
Do we need to support multiple playlists? this can allow abuse of the system.
Most likely it is better to assume only one privilege (one playlist id)